src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts

```diff
@@ -185,6 +190,9 @@ class indexes extends viewModelBase {
     }
 
     private initObservables() {
+        this.localNodeTag(this.clusterManager.localNodeTag());
```

Comment:
this.localNodeTag = this.clusterManager.localNodeTag

then you link them and avoid creating new object ;-)

Reply:
done